### PR TITLE
SISRP-39586 - Surfaces Law units on Enrollments

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.berkeley.edu/browse/SISRP-39586

A student with a Law career (or a Grad+Law with a concurrent program) can earn Law Units for a class - these are a special type of units only used by the Law school.  Law students need to see Law Units, but they should not see Points because points don't matter for Law classes.

- On Academic Summary, show the Law Units earned for each class when applicable
- Hide the Points on Law classes
- Show the total of the Units and Law Units the student took for the semester (or, if all grades for the semester are received, show the Units and Law units the student _earned_)

### CODE REVIEW

File: app/models/edo_oracle/queries.rb [hunk 1]
```diff
@@ -70,25 +70,77 @@ def self.where_course_term_updated_date(with_career_filter = true)
...
+          ENR.STDNT_ENRL_STATUS_CODE AS enroll_status,
+          ENR.WAITLISTPOSITION AS waitlist_position,
+          ENR.UNITS_TAKEN,
+          ENR.UNITS_EARNED,
+          ENR.GRADE_MARK AS grade,
+          ENR.GRADE_POINTS AS grade_points,
+          ENR.GRADING_BASIS_CODE AS grading_basis,
+          ENR.ACAD_CAREER,
+          ENR.RQMNT_DESIGNTN
         FROM SISEDO.CC_ENROLLMENTV00_VW enr
```
[the remark this answers is not in the document]
Comment (lyttam) [reply app/models/edo_oracle/queries.rb]: Thanks @marcpechaitis I forgot to change that. Will amend this PR.

**Comment by lyttam** [comment]
Incorporated @marcpechaitis ' feedback and fixed the view name.  Also added new views to the view checker (Rake task).

File: app/models/edo_oracle/queries.rb [hunk 2]
```diff
@@ -70,26 +70,78 @@ def self.where_course_term_updated_date(with_career_filter = true)
...
+
+    def self.get_term_unit_totals(person_id, academic_careers, term_id)
+      result = safe_query <<-SQL
+        SELECT 
+          SUM(SCT.TOTAL_EARNED_UNITS) AS TOTAL_EARNED_UNITS,
+          SUM(SCT.TOTAL_ENROLLED_UNITS) AS TOTAL_ENROLLED_UNITS,
+          MIN(SCT.GRADING_COMPLETE) AS GRADING_COMPLETE
+        FROM SISEDO.CLC_STUDENT_CAREER_TERMV00_VW SCT
+        WHERE SCT.CAMPUS_ID = '#{person_id}'
+        #{and_academic_career('SCT', academic_careers)}
```
[the remark this answers is not in the document]
Comment (davidlimcheng) [reply app/models/edo_oracle/queries.rb]: Unless they belong there, in which case I believe they need to be wrapped in single quotes?

[the remark this answers is not in the document]
Comment (lyttam) [reply app/models/edo_oracle/queries.rb]: @davidlimcheng actually it's string interpolation: https://ruby-operators.herokuapp.com/crab_claws

[the remark this answers is not in the document]
Comment (davidlimcheng) [reply app/models/edo_oracle/queries.rb]: @lyttam Ah, I always thought those needed to be wrapped in quotes.  Tried it out and the query did indeed pick it up.  Good to know!

**Pull request merged**